src/lib.rs

```diff
@@ -103,8 +103,7 @@ mod synch;
 mod syscalls;
 mod util;
 
-#[doc(hidden)]
-pub fn _print(args: ::core::fmt::Arguments<'_>) {
+fn _print(args: ::core::fmt::Arguments<'_>) {
```

Comment:
Since this is used in the macros, it must be accessible from other modules: https://git.rwth-aachen.de/acs/public/hermitcore/libhermit-rs/-/jobs/2199434#L260

```suggestion
pub(crate) fn _print(args: ::core::fmt::Arguments<'_>) {
```